### PR TITLE
Shopify page

### PR DESCRIPTION
**Pull request by filipbaranski**

### CODE REVIEW

File: _includes/shopify/estimate.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<section class="section estimate">
+  <div class="section_content">
+    <img class="image" src="/assets/images/sunny-suler.png" alt="">
+  </div>
+  <div class="section_desc">
+    <h2 class="section_heading">
+      Hi, my name is Marcin,<br> I am the <span class="text-secondary">CEO of Naturaily</span>
```
Comment (siaradona) [inline _includes/shopify/estimate.html]: brake this line

File: _includes/shopify/faq.html [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+        <div
+          class="accordion_content"
+          data-accordion-content
+          itemscope
+          itemprop="acceptedAnswer"
+          itemtype="https://schema.org/Answer"
+        >
+          <div itemprop="text">
+            Yes, you can freely try out Shopify for 14 days using the link below.
+            <br><a href="https://shopify-consulting.webflow.io/" target="_blank">Link</a>
```
Comment (siaradona) [inline _includes/shopify/faq.html]: don't need <br>. Add display property to <a>

File: _includes/shopify/faq.html [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+  $(document).ready(function() {
+    $('[data-process-accordion]').on('change', '[data-accordion-check]', function() {
+      const parent = $(this).parent();
+      const contentHeight = parent.children('[data-accordion-content]').height();
+      const contentPadding = parseInt(parent.children('[data-accordion-content]').css('padding-bottom'), 10)
+      const contentHeightWithPadding = contentHeight + contentPadding;
+
+      if ($(this).is(':checked')) {
+        parent.addClass('opened')
+        .animate({ 'padding-bottom' : contentHeightWithPadding }, 300)
```
Comment (siaradona) [inline _includes/shopify/faq.html]: spacing

File: _includes/shopify/services.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  <div class="section_name">
+    <span class="section_name-number">
+      02
+    </span>
+    Services
+  </div>
+  <ul class="shopify-services-list">
+    <li class="shopify-services-list-item">
+      <p class="shopify-services-list-header">Services</p>
+      <div  class="shopify-services-list-item-container">
```
Comment (siaradona) [inline _includes/shopify/services.html]: spacing

File: _includes/shopify/services.html [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        </p>
+        <p class="shopify-services-list-item-text">
+          We design the UX/UI of e-commerce stores and perform A/B tests to maximize 
+          conversion rates, ensure outstanding user experience and minimize bounce rates.
+        </p>
+      </div>
+      <img src="/assets/images/shopify/services1.png" />
+    </li>
+    <li class="shopify-services-list-item reverse">
+      <div  class="shopify-services-list-item-container">
```
Comment (siaradona) [inline _includes/shopify/services.html]: .

File: _includes/shopify/services.html [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        </p>
+        <p class="shopify-services-list-item-text">
+          We support businesses in integrating their stores' with existing ERP, payment or delivery 
+          software environments, reducing time spent on manual tasks.
+        </p>
+      </div>
+      <img src="/assets/images/shopify/services2.png" />
+    </li>
+    <li class="shopify-services-list-item">
+      <div  class="shopify-services-list-item-container">
```
Comment (siaradona) [inline _includes/shopify/services.html]: .

File: _includes/shopify/services.html [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        </p>
+        <p class="shopify-services-list-item-text">
+          Turn manual tasks, like data input or mailing, into automated activities to help you save 
+          time, go home early or focus on the most important business goals.
+        </p>
+      </div>
+      <img src="/assets/images/shopify/services3.png" />
+    </li>
+    <li class="shopify-services-list-item reverse">
+      <div  class="shopify-services-list-item-container">
```
Comment (siaradona) [inline _includes/shopify/services.html]: .

File: _sass/new-styles/modules/shopify/_about.scss [hunk 8]
```diff
@@ -0,0 +1,123 @@
+.shopify-about {
+  padding: 150px 95px 150px 160px;
+  box-sizing: border-box;
+  flex-direction: column;
+  background-color: $color-code-orange-bg;
+  &-container {
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_about.scss]: add blank line before class

File: _sass/new-styles/modules/shopify/_carousel.scss [hunk 9]
```diff
@@ -0,0 +1,110 @@
+.shopify-carousel {
+    width: 100%;
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_carousel.scss]: spacing

File: _sass/new-styles/modules/shopify/_faq.scss [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+  .accordion_content {
+    margin: 0;
+    padding: 0 100px 25px 105px;
+    width: calc(100% - 205px);
+    color: $color-global-font;
+    font-size: 16px;
+    line-height: 24px;
+    border-bottom: 1px solid transparent;
+    border-image: linear-gradient(to left, $color-secondary 0,
+                                            $color-secondary 20px,
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_faq.scss]: spacing

File: _sass/new-styles/modules/shopify/_footer.scss [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+  padding-left: 100px;
+  padding-right: 100px;
+  margin-top: 150px;
+  padding-bottom: 50px;
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  position: relative;
+  flex-direction: row;
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_footer.scss]: it is default value

File: _sass/new-styles/modules/shopify/_footer.scss [hunk 12]
```diff
@@ -0,0 +1,110 @@
+.shopify-footer {
+  padding-left: 100px;
+  padding-right: 100px;
+  margin-top: 150px;
+  padding-bottom: 50px;
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_footer.scss]: remove all paddings and add `padding: 0 100px 50px;`

File: assets/images/shopify/analytics.svg [hunk 13]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (siaradona) [inline assets/images/shopify/analytics.svg]: add icon to icomoon font and remove from assets

File: _includes/shopify/estimate.html (hunk 1 above)
Comment (filipbaranski) [reply _includes/shopify/estimate.html]: broken

File: _includes/shopify/faq.html (hunk 2 above)
Comment (filipbaranski) [reply _includes/shopify/faq.html]: changed

File: _includes/shopify/faq.html (hunk 3 above)
Comment (filipbaranski) [reply _includes/shopify/faq.html]: added

File: _includes/shopify/services.html (hunk 4 above)
Comment (filipbaranski) [reply _includes/shopify/services.html]: fixed

File: _includes/shopify/services.html (hunk 5 above)
Comment (filipbaranski) [reply _includes/shopify/services.html]: fixed

File: _includes/shopify/services.html (hunk 6 above)
Comment (filipbaranski) [reply _includes/shopify/services.html]: fixed

File: _sass/new-styles/modules/shopify/_about.scss (hunk 8 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_about.scss]: added

File: _sass/new-styles/modules/shopify/_carousel.scss (hunk 9 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_carousel.scss]: fixed

File: _sass/new-styles/modules/shopify/_faq.scss (hunk 10 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_faq.scss]: fixed

File: _sass/new-styles/modules/shopify/_footer.scss (hunk 11 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_footer.scss]: yes it is - fixed

File: _sass/new-styles/modules/shopify/_footer.scss (hunk 12 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_footer.scss]: done

File: _includes/shopify/services.html (hunk 7 above)
Comment (filipbaranski) [reply _includes/shopify/services.html]: fixed

File: _sass/new-styles/modules/shopify/_results.scss [hunk 14]
```diff
@@ -0,0 +1,162 @@
+.shopify-results {
+  padding: 150px 205px 150px 160px;
+  margin-top: 50px;
+  background-color: #FFF7F1;
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_results.scss]: `$color-white`?

File: _sass/new-styles/modules/shopify/_results.scss [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+      width: 152px;
+      height: 314px;
+      &:after {
+        width: 314px;
+        height: 314px;
+      }
+    }
+  }
+  &-list {
+    background: #FFF;
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_results.scss]: `$color-white`

File: _sass/new-styles/modules/shopify/_testimonials.scss [hunk 16]
```diff
@@ -0,0 +1,63 @@
...
+                  border-top-right-radius: 5px;
+                  border-bottom-right-radius: 5px;
+                  border-right: 1px solid rgba(59, 34, 15, 0.1);
+              }
+          }
+      }
+  }
+  > a {
+      font-size: 11px;
+      color: #FF5E45;
```
Comment (siaradona) [inline _sass/new-styles/modules/shopify/_testimonials.scss]: `$color-secondary`

File: _sass/new-styles/modules/shopify/_results.scss (hunk 14 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_results.scss]: Ale to nie jest biały

Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_results.scss]: Poprawione

File: _sass/new-styles/modules/shopify/_results.scss (hunk 15 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_results.scss]: Fixed

File: _sass/new-styles/modules/shopify/_testimonials.scss (hunk 16 above)
Comment (filipbaranski) [reply _sass/new-styles/modules/shopify/_testimonials.scss]: Fixed

File: _includes/shopify/carousel.html [hunk 17]
```diff
@@ -0,0 +1,100 @@
+
```
Comment (siaradona) [inline _includes/shopify/carousel.html]: remove blank line

**Pull request merged**